Error when upgrade to v1.2.0

I deployed calert by helm and got the error when use v1.2.0 as below:

> ERR: 2020/05/16 14:03:25 /home/karan/Projects/calert/main.go:90: Error reading config: Config File "config" Not Found in "[/]"

@mr-karan I ran into these same problems now when trying out the `v1.2.0` docker image tag.

Before, the arguments were:
[`"--config.file", "/etc/calert/config.toml"`](https://github.com/mr-karan/calert/blob/19e30bd1c7c6773ef538ad866ad13322d45b409b/docker/Dockerfile#L16)

Now they are:
[`"--config", "/etc/calert/config.toml"`](https://github.com/mr-karan/calert/blob/master/Dockerfile#L6)

Note `config.file` vs `config`.

If I run the docker image locally and compare those arguments, I see the new arguments don't work:

```bash
$ docker run --rm -it --name calert mrkaran/calert:v1.2.0 \
    ./calert --config /etc/calert/config.toml
ERR: 2020/05/20 09:26:38 /home/karan/Projects/calert/main.go:90: Error reading config: Config File "config" Not Found in "[/]"
```
```bash
$ docker run --rm -it --name calert mrkaran/calert:v1.2.0 \
    ./calert --config.file /etc/calert/config.toml
SYS: 2020/05/20 09:26:49 /home/karan/Projects/calert/main.go:134: listening on :6000 | /tmp/calert.sock
^C%
```

The helm chart (that @vuxuanlai and I are using) doesn't support overriding the command (so we don't have a workaround), but maybe it's better if we patch the [Dockerfile](https://github.com/mr-karan/calert/blob/master/Dockerfile#L6)?

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,4 +3,4 @@ RUN apk --no-cache add ca-certificates
 COPY calert /
 COPY config.sample.toml  /etc/calert/config.toml
 VOLUME ["/etc/calert"]
-CMD ["./calert", "--config", "/etc/calert/config.toml"]  
+CMD ["./calert", "--config.file", "/etc/calert/config.toml"]

